sycl/test/basic_tests/accessor/local_accessor_default_ctor.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %t.out
+
+#include <sycl/sycl.hpp>
+
+int main ()
+{
+  using AccT = sycl::local_accessor<int, 1>;
+  bool empty;
+  size_t byte_size;
+  size_t size;
+  size_t max_size;
+
+  AccT acc;
+  empty = acc.empty();
+  byte_size = acc.byte_size();
+  size = acc.size();
+  max_size = acc.max_size();
+  // The return values of get_pointer() and get_multi_ptr() are
+  // unspecified. Just check they can run without any issue.
+  auto ptr = acc.get_pointer();
+  // TODO: uncomment check with get_multi_ptr() when SYCL 2020 multi_ptr feature
+  // will be merged
+  // auto multi_ptr = acc.get_multi_ptr();
```

Comment:
multi_ptr was implemented and merged to intel/llvm. Could you please uncomment the same line in sycl/test/basic_tests/accessor/accessor_default_ctor.cpp?

Reply:
Correct me if I'm wrong, but the problem here isn't that `multi_ptr` is missing, but that `get_multi_ptr` is missing from `local_accessor`. It looks like the comment originates from an older test: sycl/test/basic_tests/accessor/accessor_default_ctor.cpp

Reply:
Oh, I see, `get_multi_ptr` is not yet implemented for the accessors.